Python/pywarpx/picmi.py

```diff
@@ -1448,12 +1449,34 @@ def laser_antenna_initialize_inputs(self, laser):
 
 
 class LoadInitialField(picmistandard.PICMI_LoadGriddedField):
+    def init(self, kw):
+        self.do_divb_cleaning_external = kw.pop('do_divb_cleaning_external', True)
```

Comment:
This should be `self.do_divb_cleaning_external = kw.pop('warpx_do_divb_cleaning_external', None)` so that it has the same default as the compiled code and has the `warpx` prefix.